apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-06-22T20:41:21Z"
-  build_hash: 38e309d07369b1e33250473084f91db3894c4379
-  go_version: go1.16.4 darwin/amd64
-  version: v0.2.3
-api_directory_checksum: 650887de24f94277e21771990d1391198cce93c3
+  build_date: "2021-06-30T16:49:27Z"
+  build_hash: 6911f924191f5315747c8e81fbcbf29b4edc8cb1
+  go_version: go1.16.4 linux/amd64
+  version: v0.3.1
+api_directory_checksum: 66483eaa00d69d0952996efb66542595709e6a6d
 api_version: v1alpha1
-aws_sdk_go_version: v1.38.11
+aws_sdk_go_version: ""
```

Comment:
To keep this version up-to-date you'll have to regenerate the controller using `AWS_SDK_GO_VERSION=v1.38.11` environment variable :)